lib/kitchen/driver/ec2.rb

```diff
@@ -79,6 +79,7 @@ class Ec2 < Kitchen::Driver::Base # rubocop:disable Metrics/ClassLength
       default_config :interface,           nil
       default_config :http_proxy,          ENV["HTTPS_PROXY"] || ENV["HTTP_PROXY"]
       default_config :retry_limit,         3
+      default_config :tenancy,             "dedicated"
```

Comment:
Not sure if this is the real default value when tenancy is not specified. I guess that by default should be "default". Guess not everybody wants to use dedicated instances.